src/components/Header.tsx

```diff
@@ -5,10 +5,14 @@ import '../styles/Header.scss';
 const logoDesktop = './images/logo-desktop.svg';
 const logoMobile = './images/logo-mobile.svg';
 const iconImportCSV = './images/importCSV.svg';
+const iconHome = './images/home.svg';
+const iconPeopleFavorite = './images/people-favorite.svg';
 
 const Header = () => {
+    const url = window.location.href.split('/');
```

Comment:
why not use 
`window.location.pathname` ?

Reply:
this is exactly perfect decision  for my case, thank you.